languages/javascript/src/main/java/de/jplag/javascript/JavaScriptLanguage.java

```diff
@@ -25,4 +25,9 @@ public String[] suffixes() {
     public String getName() {
         return "JavaScript";
     }
+
+    @Override
+    public boolean hasPriority() {
+        return false;
```

Comment:
```suggestion
        return false; // Since this module extends Typescript, it has no priority.
```

Reply:
tbh, I do not 100% understand the priority. What would happen if I'd have three languages; where 2 have priority?